### ISSUE TITLE
Getting error on build about AFHTTPClient.h

### ISSUE THREAD

**Issue by smitssmiling** [kylebrowning/drupal-ios-sdk#102]
Hi,

I just installed Cocoapods and trying to add drupal-ios-sdk in my Xcode project. I also followed all the steps that were told on the respective AFNetworking library. But when I try to build my project it gives me error - AFHTTPClient.h file not found. 
I see that its being imported in 2 files - DIOSCSRFAFHTTPClient.h and DIOSSession.h
But I dont see any AFHTTPClient.h file under AFNetworking folder under the project's Pod. 
Did anyone come across this error?
Any help would be really appreciated.

**Comment by verypossible** [comment]
http://stackoverflow.com/questions/16389888/afhttpclient-h-file-not-found

Check this question on Stack Overflow -- I followed this and fixed a similar issue.

**Comment by smitssmiling** [comment]
Hi, Thank you for helping. I realized that I was using 2.0 Version of AFNetworking library. But the latest version of drupal-ios-sdk is compatible with older versions of AFNetworking ( version 1.x and older ). The newer version of AFNetworking library (2.0) doesn't have AFHTTPClient.h class, and since was the error. Check out the Requirements section - https://github.com/AFNetworking/AFNetworking#requirements of AFNetworking before using it.
So long story short, I installed the pod for my xcode project with 1.x AFNetworking version and waala..!!! No more errors.

**Comment by kylebrowning** [comment]
2.0 has been added to the project now.
